code/game/gamemodes/game_mode_latespawn.dm

```diff
@@ -6,7 +6,7 @@
 ///process()
 ///Called by the gameticker
 /datum/game_mode/process()
-	if(round_autoantag && world.time >= next_spawn && !emergency_shuttle.departed)
+	if(round_autoantag && world.time >= next_spawn && emergency_shuttle.shuttle.moving_status == SHUTTLE_IDLE)
```

Comment:
This is not a fix at all. SHUTTLE_IDLE only stops the crew from being traitored while the shuttle is flying to the station, or back to CC. It isn't going to stop them from spawning as antag when:
* The first 6 minutes of the shuttle timer have elapsed.
* The shuttle is docked at the station.

Reply:
AAAAAAAAAAAAAUGH okay. How would one fix that.